classes/simplemap.php

```diff
@@ -2550,7 +2550,7 @@ function get_default_shortcode_atts() {
 			}
 
 			$atts = $tax_atts + array(
-					'search_title'         => __( 'Find Locations Near:', 'simplemap' ),
+					'search_title'         => __( 'Find Locations Near:', 'SimpleMap' ),
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->